after/ftplugin/xml.vim

```diff
@@ -1,12 +1,6 @@
 if gradle#isGradleProject()
-
-  call gradle#setClassPath()
-  call gradle#setupGradleCommands()
-
   if android#isAndroidProject()
-    call android#setAndroidSdkTags()
-    call android#setClassPath()
-    call android#setupAndroidCommands()
+    autocmd BufEnter *.xml XMLns android
+    autocmd BufEnter AndroidManifest.xml XMLns manifest
```

Comment:
In that case I would prefer something like:

    if android#isAndroidProject()
         " load classpaths.....
         if expand('%:t') == 'AndroidManifest.xml'
              XMLns manifest
         else
              XMLns android
         endif
    endif

Is it really necessary to have two different xml data files? Based on context they both could work in a single one without interfering with each other right?